### PR TITLE
Updated linear networks to v0.17.1

### PR DESCRIPTION
**Pull request by semercim**
*Değişikliklerin açıklaması:*

Bu çekme talebini göndererek, bu katkıyı tercih ettiğiniz şartlar 
altında kullanabileceğinizi, değiştirebileceğinizi, kopyalayabileceğinizi 
ve yeniden dağıtabileceğinizi onaylıyorum.

### CODE REVIEW

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md [hunk 1]
```diff
@@ -195,7 +195,7 @@ $$\begin{aligned}f(u, v) & = (u+v)^{2} \\u(a, b) & = (a+b)^{2}, \qquad v(a, b) =
 
 Sinir ağları ile çalışırken bu tür denklem zincirleri yaygındır, bu nedenle bu tür işlevlerin gradyanlarının nasıl hesaplanacağını anlamaya çalışmak çok önemlidir. Hangi değişkenlerin doğrudan birbiriyle ilişkili olduğuna bakarsak, bu bağlantının görsel ipuçlarını :numref:`fig_chain-1`'de görmeye başlayabiliriz.
 
-![Düğümlerin değerleri temsil ettiği ve kenarların işlevsel bağımlılığı gösterdiği yukarıda geçen işlev ilişkileri.](../img/ChainNet1.svg)
+![Düğümlerin değerleri temsil ettiği ve kenarların işlevsel bağımlılığı gösterdiği yukarıda geçen işlev ilişkileri.](../img/chain-net1.svg)
```
Comment (AnirudhDagar) [inline chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:198]: This svg image is not present. In the english repo it got renamed a while back. See [commit](https://github.com/d2l-ai/d2l-en/commit/3e9358109ce349efb2fd6041f3cc496d4bc1eab2).

You need to go to the /img directory and rename the file `ChainNet.svg` to `chain-net1.svg` for this to work.
Simply a `git mv ChainNet.svg chain-net1.svg` should work for renaming.

File: chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md [hunk 2]
```diff
@@ -236,7 +236,7 @@ Sürecin anlamını düşünmekte fayda var. $f(u (a, b), v(a, b))$ biçimindeki
 
 Sağdaki işlevlerin soldakilere, şekilde gösterildiği gibi, bağlı oldukları farklı bir işlev ağına sahip olduğumuzu düşünün :numref:`fig_chain-2`.
 
-![Zincir kuralının daha ince bir başka örneği.](../img/ChainNet2.svg)
+![Zincir kuralının daha ince bir başka örneği.](../img/chain-net2.svg)
```
Comment (AnirudhDagar) [inline chapter_appendix-mathematics-for-deep-learning/multivariable-calculus.md:239]: Same as above.

**Review by AnirudhDagar — COMMENTED** [review]
Hi @semercim the CI fails due to some svg files it cannot find. Please look at my comment for a suggested fix. Let me know if that works, then we can go ahead with the merge after everything is green!

**Comment by semercim** [comment]
Hi @AnirudhDagar,

I had already made these changes with commit a99a314103a7c4783319732d5ecc87b8690cc83f in my feature branch. I also have the svg image files in the img folder. I checked them again, but I do not think the issues are related to these two specific images files (I keep both versions: chain-net1&2 and ChainSet1&2) in the multivariable-calculus md file. They look OK.

I also checked whether I have a missing svg file that is referred from the appendix for maths chapter but I could not detect any missing svg issue.

I would be glad if you could have a deeper look into this problem to help me. Thanks,

**Comment by semercim** [comment]
Hi @AnirudhDagar,

I had already made these changes with commit a99a314103a7c4783319732d5ecc87b8690cc83f in my feature branch. I also have the svg image files in the img folder. I checked them again, but I do not think the issues are related to these two specific images files (I keep both versions: chain-net1&2 and ChainSet1&2) in the multivariable-calculus md file. They look OK.

I also checked whether I have a missing svg file that is referred from the appendix for maths chapter but I could not detect any missing svg issue.

I would be glad if you could have a deeper look into this problem to help me. Thanks,

**Comment by AnirudhDagar** [comment]
I'm looking into the problem. On a side note, are the changes to `Jenkinsfile` intentional?

**Comment by mli** [comment]
Job d2l-tr/PR-40/13 is complete. 
Check the results at http://preview.d2l.ai/d2l-tr/PR-40/

**Comment by AnirudhDagar** [comment]
Should be fixed now!

**Comment by mli** [comment]
Job d2l-tr/PR-40/14 is complete. 
Check the results at http://preview.d2l.ai/d2l-tr/PR-40/

**Comment by semercim** [comment]
> Should be fixed now!

Thanks @AnirudhDagar . Everything seems all right now. Merging the branch and closing the PR.

**Pull request merged**